src/providers/Auth.tsx

```diff
@@ -45,16 +42,39 @@ export const AuthProvider = ({ children }: any) => {
 export const useAuth = () => {
   const [
     [token, setTokenState],
-    [permissions, setPermissions]
   ] = useContext(AuthContext);
 
-  const getPermissions = () => {
-    return permissions;
+  const getPermissions = (token?: string) => {
+    if (token) {
+      return getPermissionsFromToken(token) || [];
+    }
+
+    return getPermissionsFromToken(getToken()) || [];
+  };
+
+  const canRead = (token?: string) => {
+    if (token) {
+      return getPermissionsFromToken(token).filter(permission => permission.toLowerCase().startsWith(`${RDPC_DOMAIN}-${RDPC_REGION}.READ`.toLowerCase())).length > 0;
```

Comment:
`${RDPC_DOMAIN}-${RDPC_REGION}` seems to be quite repeated, should we just strip it out to a constant somewhere?